style.php

```diff
@@ -34,8 +34,8 @@
         padding: 0;
     }
     .wp-caption-text {
-        font-size: 12px;
-        line-height: 1.5em;
+        font-size: .75rem;
+        line-height: 1.4;
```

Comment:
Do you need to specify a unit here? `em`?

Reply:
No, it becomes relative to the font size if you don't specify: .75rem=12px, 1.4*12px=16.8px.